src/globus_sdk/experimental/globus_app/globus_app.py

```diff
@@ -441,11 +440,11 @@ class ClientApp(GlobusApp):
     A ``GlobusApp`` using client credentials - useful for service accounts and
     automation use cases.
 
-    ``ClientApp``s are always used with confidential clients either by passing
+    ``ClientApp`` s are always used with confidential clients either by passing
```

Comment:
The 's' should still be attached; does it not render correctly when placed next to it? If not, perhaps a backslash is the solution here, though that may require this docstring to convert to an r-string (`r"""`).

```suggestion
    ``ClientApp``\s are always used with confidential clients either by passing
```

Reply:
Yep this was a rendering issue; ``ClientApp``\\s solved it